package.json

```diff
@@ -1,18 +1,19 @@
 {
   "private": true,
-  "name": "zendeskgarden-website",
+  "name": "@zendeskgarden/website",
   "version": "0.0.0",
   "scripts": {
     "build": "gatsby build",
     "clean": "gatsby clean",
-    "format": "prettier --loglevel warn --write \"**/*.{js,jsx,ts,tsx,json,md,yml}\"",
+    "format": "prettier-package-json --write && yarn format:all --write",
+    "format:all": "prettier --loglevel warn '**/*.{js,jsx,ts,tsx,json,md,yml}' '!.github/ISSUE_TEMPLATE.md'",
     "lint": "yarn lint:js && yarn lint:css && yarn lint:md",
-    "lint:css": "stylelint \"src/**/*.{js,ts,tsx}\"",
+    "lint:css": "stylelint 'src/**/*.{js,ts,tsx}'",
     "lint:js": "eslint ./ --ext js,ts,tsx --max-warnings 0",
     "lint:md": "markdownlint ./*.md src/**/*.md",
-    "serve": "gatsby serve",
-    "start": "gatsby develop",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+    "prepare": "yarn build",
```

Comment:
This site doesn't follow the standard pre-post script process as versioned packages. Why do we need to build the fulls site for this event?

Reply:
I actually want this repo to follow the same conventions as the other Garden repos. You run `yarn` and everything is ready to go.

Reply:
The prepare/build step is required in the other repos due to the lerna boostrap process.

Since build is unrelated to being able to develop the site I would lean toward it not being added here.

Reply:
I don't see it that way. Every other Garden repo (eslint-config, stylelint-config, svg-icons, etc) follows this convention. That is, once `yarn` is run, everything is intact for `yarn start`, etc. When lifecycle scripts aren't necessary, then run `yarn --ignore-scripts`.

Reply:
The approach simplifies the documentation and consistency for anyone to get up-and-started for website contribution.

Reply:
> The approach simplifies the documentation and consistency for anyone to get up-and-started for website contribution.

Should the approach be geared towards maintainers since external involvement could be limited to filing issues? Seems like it would tradeoff consistency between repositories, but maybe that might not be a bad thing since the nature of this repo is a bit different? This also seems like a topic that could be reassessed as the project evolves.

Reply:
Yes. And the maintainers for `website` include both eng and non-eng individuals. We must make the entry process super simple – and consistent with the other repos.